core/src/main/scala/chisel3/CompileOptions.scala

```diff
@@ -82,7 +82,30 @@ object ExplicitCompileOptions {
     explicitInvalidate = false,
     inferModuleReset = false
   ) {
+    override def migrateInferModuleReset = false
+
     override def emitStrictConnects = false
+
```

Comment:
```suggestion
    override def migrateInferModuleReset = false
    override def emitStrictConnects = false
```

make whitespace consistent